lib/apress/page_info/version.rb

```diff
@@ -1,5 +1,6 @@
+# frozen_string_literal: true
 module Apress
   module PageInfo
-    VERSION = '0.5.0'.freeze
+    VERSION = '0.6.0'.freeze
```

Comment:
Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.